package-lock.json

```diff
@@ -45486,8 +45473,7 @@
             }
         },
         "url-parse": {
-            "version": "1.5.10",
-            "resolved": "https://registry.npmjs.org/url-parse/-/url-parse-1.5.10.tgz",
+            "version": "https://registry.npmjs.org/url-parse/-/url-parse-1.5.10.tgz",
```

Comment:
This change seems off.

Reply:
checking

Reply:
Funky. Don't think I've ever seen that before.

Reply:
I had `npm -v 8.5.5`. I `nvm use v16` rather than installing `16.13.0`. Now I'm using the right version of Node (16.13.0) and NPM (8.1.0) 🙂